manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -266,7 +286,7 @@ func (r *FybrikApplicationReconciler) deleteExternalResources(applicationContext
 		return nil
 	}
 
-	r.Log.V(0).Info("Reconcile: FybrikApplication is deleting the generated " + applicationContext.Status.Generated.Kind)
+	r.Log.Trace().Str(logging.FYBRIKAPPUUID, utils.GetFybrikApplicationUUID(applicationContext)).Str(logging.ACTION, logging.DELETE).Msg("Reconcile: FybrikApplication is deleting the generated " + applicationContext.Status.Generated.Kind)
```

Comment:
If we register the action (delete, update, create), perhaps we could add Kind as well and reduce this long message in this case, i.e. DELETE Plotter, UPDATE FybrikApplication, etc.